### ISSUE TITLE
`node/_tools/test.ts`: TS2538 [ERROR] and AssertionError

### ISSUE THREAD

**Issue by wafuwafu13** [denoland/deno_std#1319]
**Describe the bug**
I want to try https://github.com/denoland/deno_std/issues/911 but tests that already exist failed because of TS2538 [ERROR] and AssertionError.

**Steps to Reproduce**
1. Set up `node/_tools/config.json` (try only `test-event-emitter-add-listeners.js`)
```json
{
  "nodeVersion": "15.5.1",
  "ignore": {
    "common": [
      "index.js"
    ],
    "parallel": [
    ]
  },
  "tests": {
    "parallel": [
      "test-event-emitter-add-listeners.js"
    ]
  },
  "suitesFolder": "suites",
  "versionsFolder": "versions"
}
```
2. Run `$ deno run --allow-read --allow-net --allow-write node/_tools/setup.ts` (manually decompress related: https://github.com/denoland/deno_std/issues/1317)
3. Run `$ deno test --allow-read --allow-run node/_tools/test.ts`
<details>
<summary>Result</summary>

```zsh
~/desktop/deno_std
$ deno test --allow-read --allow-run node/_tools/test.ts
running 1 test from file:///Users/tagawahirotaka/Desktop/deno_std/node/_tools/test.ts
error: TS2538 [ERROR]: Type 'symbol' cannot be used as an index type.
      let listeners = this._events[eventName];
                                   ~~~~~~~~~
    at file:///Users/tagawahirotaka/Desktop/deno_std/node/events.ts:99:36

TS2538 [ERROR]: Type 'symbol' cannot be used as an index type.
        this._events[eventName] = listeners;
                     ~~~~~~~~~
    at file:///Users/tagawahirotaka/Desktop/deno_std/node/events.ts:102:22

TS2538 [ERROR]: Type 'symbol' cannot be used as an index type.
      this._events[eventName] = listener;
                   ~~~~~~~~~
    at file:///Users/tagawahirotaka/Desktop/deno_std/node/events.ts:111:20

...

TS2538 [ERROR]: Type 'symbol' cannot be used as an index type.
    return this._events && Boolean(this._events[eventName]);
                                                ~~~~~~~~~
    at file:///Users/tagawahirotaka/Desktop/deno_std/node/events.ts:644:49

Found 12 errors.
test suites/parallel/test-event-emitter-add-listeners.js ... FAILED (4302ms)

failures:

suites/parallel/test-event-emitter-add-listeners.js
AssertionError: Values are not equal:


    [Diff] Actual / Expected


-   1
+   0

    at assertEquals (file:///Users/tagawahirotaka/Desktop/deno_std/testing/asserts.ts:271:9)
    at fn (file:///Users/tagawahirotaka/Desktop/deno_std/node/_tools/test.ts:40:7)
    at async asyncOpSanitizer (deno:runtime/js/40_testing.js:35:9)
    at async resourceSanitizer (deno:runtime/js/40_testing.js:72:7)
    at async exitSanitizer (deno:runtime/js/40_testing.js:99:9)
    at async runTest (deno:runtime/js/40_testing.js:215:7)
    at async Object.runTests (deno:runtime/js/40_testing.js:283:22)
    at async file:///Users/tagawahirotaka/Desktop/deno_std/1fa6aac2-3d74-4750-8738-c69deb900150$deno$test.js:4:1

failures:

        suites/parallel/test-event-emitter-add-listeners.js

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out (4356ms)
```

Related in `node/events.ts`
![スクリーンショット 2021-09-28 18 32 39](https://user-images.githubusercontent.com/50798936/135062271-57020004-7400-4756-bf28-ebca1792061c.png)
</details>

**Expected behavior**
Test passes

 https://github.com/denoland/deno_std/pull/1315 is working fine, so probably something is wrong with my development environment...?

**Environment**

```zsh
$ sw_vers
ProductName:    macOS
ProductVersion: 11.6
BuildVersion:   20G165

$ deno --version
deno 1.13.2 (release, x86_64-apple-darwin)
v8 9.3.345.11
typescript 4.3.5
```

**Comment by wafuwafu13** [comment]
@kt3k 
Question: How do I check `std version`? (Related: https://github.com/denoland/deno_std/pull/1318)

**Comment by kt3k** [comment]
The test should be passing with the latest deno. What happens if you upgrade to the latest deno? `deno upgrade` upgrades to the latest

> Question: How do I check std version? (Related: #1318)

That only applies if you use the published version of deno like the below:

```
import { listenAndServe } from "https://deno.land/std@0.108.0/http/server.ts";
```

In the above case, std version is `0.108.0`

**Comment by wafuwafu13** [comment]
>What happens if you upgrade to the latest deno?

Tests passed! Thank you!
I'll add this information to the documentation(https://github.com/denoland/deno_std/blob/main/node/README.md#setting-up-the-test-runner).

>That only applies if you use the published version of deno like the below:

I understand :+1:

**Comment by kt3k** [comment]
This should be fixed by #1325
